test/RadworksDripsGovernance.t.sol

```diff
@@ -118,6 +164,34 @@ abstract contract RadworksDripsGovernance is ProposalTest {
     // Ensure the admin role has been renounced
     assertEq(drips.admin(), address(0));
   }
+
+  function test_proposeNewAdminViaGovernance(address _newAdmin) public {
```

Comment:
Name should be `testFuzz_ProposaNewAdminViaGovernance`

Reply:
Done: https://github.com/ScopeLift/radworks-governor-upgrade/pull/21/commits/ec11571ab8c1595c882aab4119eb39672a50a00a